boards/arm/mimxrt1060_evk/pinmux.c

```diff
@@ -308,7 +308,7 @@ static int mimxrt1060_evk_init(const struct device *dev)
 #endif
 
 #if DT_NODE_HAS_STATUS(DT_NODELABEL(usdhc1), okay) && CONFIG_DISK_DRIVER_SDMMC
-	mimxrt1060_evk_usdhc_pinmux(0, true, 2, 1);
+	mimxrt1060_evk_usdhc_pinmux(0, true, 0, 7);
```

Comment:
Interesting change?

Reply:
This is based off of the way that the SDK board configuration sets up speed and drive strength initially. In the SDK, there is this snippet to determine the strength/drive speed:

```
if (freq <= 50000000)
    {
        speed    = 0U;
        strength = 7U;
    }
    else if (freq <= 100000000)
    {
        speed    = 2U;
        strength = 7U;
    }
    else
    {
        speed    = 3U;
        strength = 7U;
    }
```
I pulled that code into the soc10xx.c file, since the interface we have to the usdhc_pinmux functions doesn't allow you to pass a frequency. This change just increases the drive strength and decreases the speed of the pins at boot, since SD card negotiation starts at a low frequency, and the snippet above suggests that you should start the pins at a low speed and high strength